package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "enginn-api",
+  "version": "0.0.1",
```

```suggestion
  "version": "0.1.0",
```
https://semver.org/lang/fr/#comment-dois-je-g%C3%A9rer-les-r%C3%A9visions-dans-la-phase-initiale-de-d%C3%A9veloppement-0yz-